### ISSUE TITLE
Google Chrome window is not opened on Wits start

### ISSUE THREAD

**Issue by needz** [Samsung/Wits#25]
When running a Web application on Tizen 5.0 TV using Wits, I have the following error in the terminal, even though the application is running fine and I can edit the code in real time.

Node.js v8.12.0, macOS Catalina

How can this be fixed?

```
(node:6933) UnhandledPromiseRejectionWarning: Error
    at new LauncherError (/Users/user/Wits/node_modules/chrome-launcher/dist/utils.js:35:22)
    at new ChromeNotInstalledError (/Users/user/Wits/node_modules/chrome-launcher/dist/utils.js:66:9)
    at Launcher.<anonymous> (/Users/user/Wits/node_modules/chrome-launcher/dist/chrome-launcher.js:145:27)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/user/Wits/node_modules/chrome-launcher/dist/chrome-launcher.js:9:58)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:6933) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:6933) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
a user connected
```

**Comment by pwsses** [comment]
Hi 
It looks like npm module error. 
Wits use chrome-launcher module when launch chrome browser. 
As your log which is "at new ChromeNotInstalledError (/Users/user/Wits/node_modules/chrome-launcher/dist/utils.js:66:9)" 
chrome-launcher checked that chrome was not installed on your PC.
one more check your env.
thank you

**Comment by needz** [comment]
@pwsses Chrome is definitively installed and can be run by TizenStudio. Is there any file I can edit to set the Google Chrome location?

**Comment by pwsses** [comment]
As chrome-launcher spec, it setting chromePath when launch. But Wits support default path.
please refer 
https://github.com/GoogleChrome/chrome-launcher#readme
Wits also will review the support plan for it.

**Comment by needz** [comment]
@pwsses I have fixed it by installing the latest chrome-launcher version manually

`npm install chrome-launcher@0.12.0`

I think you should include the updated version of chrome-launcher with Wits.

But after launching Wits, I still only see a blank page with "Inspectable WebContents" and a "Wits" link, which doesn't go anywhere and the web inspector doesn't work. Please look into this, since now the tool is not usable.

**Comment by pwsses** [comment]
How about click Wits link? than you can see inspector page about Wits and your application.
your application located iframe tag in Wits application. 
please refer Debugging with google chrome DevTools in Wits Readme.
(https://github.com/Samsung/Wits)
thank you

**Comment by needz** [comment]
@pwsses When I click the link, I just get a blank page with the following error in the inspector

```
inspector.js:2984 Uncaught TypeError: document.registerElement is not a function
    at Object.UI.registerCustomElement (inspector.js:2984)
    at inspector.js:2996
    at inspector.js:2998
UI.registerCustomElement @ inspector.js:2984
(anonymous) @ inspector.js:2996
(anonymous) @ inspector.js:2998
```

**Comment by pwsses** [comment]
@needz What is inspector.js file? It is not Wits project file. it looks like script error.

**Comment by needz** [comment]
@pwsses I have made a video capture, so you understand what I mean, [here is the link](http://mvv.lt/media/wits.mov)

**Comment by pwsses** [comment]
@needz Thank you for your issue report. I also check chrome browser.

**Comment by pwsses** [comment]
@needz  I have one more question for debugging.
Would you share another browser which is work. what kind of browser? If it is a different version of chrome, please share the version info.
And do you know Tizen TV firmware info?
thank you for your help.

**Comment by needz** [comment]
@pwsses Tizen 5.0 T-MSLDEUC-1335.0

Chrome Version 81.0.4000.3 (Official Build) dev (64-bit) - doesn't work
Safari Version 13.0.4 (15608.4.9.1.3) - doesn't work
Safari Technology Preview Release 97 (Safari 13.1, WebKit 15609.1.11.4) - doesn't work
Firefox 72.0.1 (64-bit) - doesn't work

Chromium Version 79.0.3945.88 (Developer Build) (64-bit) - works
Opera Version:65.0.3467.78 - works

**Comment by pwsses** [comment]
@needz Thanks a million!! I will report to our count part with your information.

**Comment by pwsses** [comment]
@needz Sorry for late replay, We found issue over Chromium 80 version. 
We recommend temporarily using version 79 or less(https://www.slimjet.com/chrome/google-chrome-old-version.php ) and will share it again as soon as the issue is resolved.
Thank you.

**Comment by prozanne** [comment]
For your information,
We've found the way to avoid the inspector issue (over Chromium 80 version has a problem).
I added an Chrome option on WITs, and then I released (v.2.1.1)

Thanks.

**Comment by xoxefdp** [comment]
i got same issue reported by https://github.com/Samsung/Wits/issues/25#issuecomment-571952984

Chromium 88.0.4324.96 snap (also with compiled version, no snap)

![Área de trabajo 1_085](https://user-images.githubusercontent.com/4589403/105845960-cd31c680-5fdb-11eb-9578-49c417f8d520.png)

![Selección_086](https://user-images.githubusercontent.com/4589403/105846087-f8b4b100-5fdb-11eb-8e04-b30bbb269f8b.png)

**Comment by prozanne** [comment]
Hi,

Sorry for late reply.

I've tested on v88.0.4324.190(chromium), and It seems ok.
Could you please check the version of WITs?
We had a similar issue before. and the bug was fixed over v2.1.1

This option value was added for fixing this issue.
https://github.com/Samsung/Wits/blob/master/lib/appLaunchHelper.js#L154

**Comment by zemlanin** [comment]
All three runtime features were entirely removed from Chrome in March 2021, so browser ignores `--enable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports` https://chromium.googlesource.com/chromium/src/+/46ff0c3d013b0a96199e4857bce4fa06f771326b%5E%21/

Bundled-in version of DevTools (`devtools://devtools/bundled/inspector.html`) seems to work okay. Is there a way to load _it_ instead of a page served from TV IP?
